src/Cardano/Types/Value.purs

```diff
@@ -47,16 +47,16 @@ module Cardano.Types.Value
 
 import Prelude hiding (join)
 
-import Control.Alt ((<|>))
-import Control.Alternative (guard)
-import Data.Argonaut
-  ( class DecodeJson
-  , class EncodeJson
-  , JsonDecodeError(TypeMismatch)
-  , caseJsonObject
-  , encodeJson
+import Aeson
+  ( class DecodeAeson
+  , class EncodeAeson
+  , JsonDecodeError(..)
```

Comment:
Explicit constructor imports. It seems like everywhere `JsonDecodeError` is being imported like this. I'm not going to add a comment in each module, but please fix each of them